files/build_templates/sonic_debian_extension.j2

```diff
@@ -54,17 +54,17 @@ sudo cp $IMAGE_CONFIGS/environment/motd $FILESYSTEM_ROOT/etc/
 sudo mkdir -p $FILESYSTEM_ROOT/etc/sonic/
 sudo mkdir -p $FILESYSTEM_ROOT/usr/share/sonic/templates/
 
-# Install dependencies for SONiC config engine
-# TODO: pip-install instead of apt-get after config engine wrapped into a wheel, even better use pip implicitly installing dependencies
+# Install dependencies for SONiC config engine 
 sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install \
+    python-dev     \
```

Comment:
It's in base image. Do we also care a lot about size here? I prefer we keep it for easier usage of pip.

---
In reply to: [105694773](https://github.com/Azure/sonic-buildimage/pull/388#discussion_r105694773) [](ancestors = 105694773)